packages/api/src/external/carequality/patient.ts

```diff
@@ -31,26 +33,80 @@ const context = "cq.patient.discover";
 const iheGateway = makeIheGatewayAPIForPatientDiscovery();
```

Comment:
shouldn't we define this in `validatePDEnabledAndInitGW()` - if that's the entry point to using it?

Reply:
yea good point updating